src/express-cli.js

```diff
@@ -645,14 +646,14 @@ export async function cli() {
       '⛔ This command is only available for non-dockerized devnets. Make sure to target such environment...'
     )
     await rpcTest()
-  } else if (options.fundGanacheAccounts) {
+  } else if (options.fundAnvilAccounts) {
     console.log('📍Command --fund-ganache-accounts')
```

Comment:
let's also change this log

Reply:
Also, let's change the 
```
.option('-fga, --fund-ganache-accounts', 'Add funds to ganache accounts')
```
At line 167 in `express-cli.js` and the relative command description in the `README`